docs/README.md

```diff
@@ -1,76 +1,137 @@
+can be versioned easily. However, in our experience it is not that easy to create custom diagrams
+and animations using LaTeX, mainly because of its declarative nature. If you can speak in
+[TikZ](https://www.overleaf.com/learn/latex/TikZ_package) and you understand the error messages of
+`pdflatex`, you are probably fine. If not, creating slides with complex animations, diagrams and
+source code snippets might be easier for you in Python.
+
+    While *Elsie* also has basic support for rendering [LaTeX](userguide/latex.md), if your
+    presentation is mostly composed of math formulas, it might be easier to create it in LaTeX
+    directly.
+
+- **Reveal.js**
+[reveal.js](https://revealjs.com) is a great tool for making HTML presentations. It supports syntax
+highlighting, has good-looking animations and can be versioned. However, it shares some disadvantages
+of LaTeX/Beamer, which stems from the fact that it is also declarative. Creating an animation that
+would walk line-by-line through a source code snippet or that would repeatedly display and hide
+objects in custom fragments is difficult. You can actually use JavaScript to create more complex
+animations, but it's not integrated very well, and the library itself does not offer you any API
+to make it easier.
+
+    That being said, if you are fine with declarative description of slides, and you prefer
+    HTML/CSS to Python/SVG, I would recommend you to use `reveal.js`.
+
+- **Prezi**
+If you want presentations with three or more dimensions, it's a good choice. Otherwise, the
+disadvantages of PowerPoint also apply here.
+
+None of the above solutions allow you to build slides using (an imperative) programming language,
+which gives you a number of useful powers. You can split a large presentation into several
+modules/files, parametrize animations or objects that appear often using functions, create
+arbitrarily complex slides and animations or interactively modify the font, aspect ratio or text
+color/size of your whole presentation by changing a single line of code. This is the ultimate
+feature of *Elsie* that is not available in the other tools.
 
 ## FAQ
-- Why do you use `Inkscape` instead of e.g. `Cairo`?
-TODO
-- Why don't you use a known layout model, e.g. `flexbox`?
-TODO
-- Why is there no support for lists in `Elsie`?
-TODO
+- Why do you use `Inkscape` instead of e.g. `Cairo` for rendering slides?
+We are experimenting with a `Cairo` backend, but it would be probably difficult to fully support
+embedding SVG images into slides with Cairo, which is something that we use a lot. But it is
+possible that *Elsie* will not require `Inkscape` sometime in the future.
+- Why don't you use an existing layout model, e.g. `flexbox`?
+TODO: ???
+Also, we haven't found a usable binding of a good and standalone layout model in Python. If you
+know of any, please let us know.
+- Why is there no built-in support for lists (bullet items etc.) in `Elsie`?
```

Comment:
Just idea: Maybe mention here that we want to present complete and compact "building blocks" and you can easily build lists from this raw materials